### ISSUE TITLE
Trame app crashes with large meshes

### ISSUE THREAD

**Issue by leilakanync** [pyvista/pyvista#4708]
### Describe the bug, what's wrong, and what you expected.

Hi,

I was trying to display a large mesh with the new trame backend (~5M vertices) in a jupyter notebook, but the app crashes with this error.

```
ERROR:root:Exception raised
ERROR:root:ConnectionResetError(104, 'Connection reset by peer')
ERROR:root:Traceback (most recent call last):
  File "/home/container_user/.local/lib/python3.10/site-packages/wslink/protocol.py", line 329, in onMessage
    await self.sendWrappedMessage(
  File "/home/container_user/.local/lib/python3.10/site-packages/wslink/protocol.py", line 466, in sendWrappedMessage
    await ws.send_bytes(attachments[key])
  File "/usr/local/lib/python3.10/dist-packages/aiohttp/web_ws.py", line 315, in send_bytes
    await self._writer.send(data, binary=True, compress=compress)
  File "/usr/local/lib/python3.10/dist-packages/aiohttp/http_websocket.py", line 688, in send
    await self._send_frame(message, WSMsgType.BINARY, compress)
  File "/usr/local/lib/python3.10/dist-packages/aiohttp/http_websocket.py", line 659, in _send_frame
    await self.protocol._drain_helper()
  File "/usr/local/lib/python3.10/dist-packages/aiohttp/base_protocol.py", line 90, in _drain_helper
    await asyncio.shield(waiter)
  File "/usr/lib/python3.10/asyncio/futures.py", line 285, in __await__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.10/asyncio/tasks.py", line 304, in __wakeup
    future.result()
  File "/usr/lib/python3.10/asyncio/futures.py", line 201, in result
    raise self._exception.with_traceback(self._exception_tb)
  File "/usr/lib/python3.10/asyncio/selector_events.py", line 949, in _write_ready
    n = self._sock.send(self._buffer)
ConnectionResetError: [Errno 104] Connection reset by peer
```

However it works well with pythreejs.
I even have the issue with 1.5M vertices

### Steps to reproduce the bug.

Call the plot function on a pyvista object having around 5 millions vertices.

### System Information

```shell
--------------------------------------------------------------------------------
  Date: Wed Jul 26 15:56:10 2023 Europe

                  OS : Linux
              CPU(s) : 8
             Machine : x86_64
        Architecture : 64bit
                 RAM : 54.9 GiB
         Environment : Python
          GPU Vendor : Mesa/X.org
        GPU Renderer : llvmpipe (LLVM 12.0.0, 256 bits)
         GPU Version : 4.5 (Core Profile) Mesa 21.2.6
    MathText Support : False

  Python 3.10.12 (main, Jun  7 2023, 12:45:35) [GCC 9.4.0]

             pyvista : 0.40.0
                 vtk : 9.2.6
               numpy : 1.23.5
          matplotlib : 3.7.1
              scooby : 0.7.2
               pooch : v1.7.0
             imageio : 2.27.0
             IPython : 7.16.1
            colorcet : 3.0.1
             cmocean : v3.0.3
          ipyvtklink : 0.2.2
          ipywidgets : 7.6.3
               scipy : 1.11.0
                tqdm : 4.65.0
              meshio : 5.3.4
          jupyterlab : 4.0.3
           pythreejs : 2.4.2
               trame : 2.5.0
        trame_client : 2.9.4
        trame_server : 2.11.5
           trame_vtk : 2.5.4
jupyter_server_proxy : 4.0.0
        nest_asyncio : 1.5.6
--------------------------------------------------------------------------------
```

### Screenshots

![Screenshot 2023-07-26 at 17 59 55](https://github.com/pyvista/pyvista/assets/130442217/04b0e342-a438-4435-a2d6-fdf62b75011e)

**Comment by banesullivan** [comment]
This looks like a duplicate of #3979 which is due to connection timeouts during the transfer of the large data through `jupyter-server-proxy`. Please keep an eye on #5001 as we'll soon transition away from using `jupyter-server-proxy` to the Jupyter comms directly which should solve this.

**Comment by leilakanync** [comment]
Indeed. Thank you for your answer, I will keep an eye on the issue!

**Comment by jourdain** [comment]
Should be fixed when using `trame>=3.6` which should update `trame-[client|server]>3` and `wslink>2.`
